### PR TITLE
ko: Fixup compat/spec sections

### PR DESCRIPTION
**Pull request by queengooborg**
This PR is a sort-of follow-up to #11691.  In this PR, a few adjustments are made to the spec and compatibility data sections:

- Any English "Specifications" header was replaced with "명세"
- Any English "Browser Compatibility" header was replaced with "브라우저 호환성"
- Browser compat sections with multiple `{{Compat}}` macro calls were compressed into a single call (since the macro does this for us now)

### CODE REVIEW

File: files/ko/web/api/web_storage_api/index.md [hunk 1]
```diff
@@ -49,12 +49,6 @@ Web Storage의 두 가지 방식은 다음과 같습니다.
 
 ## 브라우저 호환성
 
-### `Window.localStorage`
-
-{{Compat}}
-
-### `Window.sessionStorage`
```
Comment (hochan222) [inline files/ko/web/api/web_storage_api/index.md:56]: Previously, just did `{{Compat}}` conversion, but I didn't expect this to happen.

File: files/ko/web/css/_doublecolon_-webkit-scrollbar/index.md [hunk 2]
```diff
@@ -91,32 +91,6 @@ Who thought scrollbars could be made weeeeird?
...
-### `::-webkit-scrollbar-track-piece`
-
-{{Compat}}
-
-### `::-webkit-scrollbar-corner`
-
-{{Compat}}
-
-### `::-webkit-resizer`
-
```
Comment (hochan222) [inline files/ko/web/css/_doublecolon_-webkit-scrollbar/index.md:119]: There are cases like this.. I'll have to be careful when writing regex in the future.

**Review by hochan222 — APPROVED** [review]
Sorry for merging in [the last PR](https://github.com/mdn/translated-content/pull/11691#discussion_r1111375300) without asking. (+ I've looked all over... But didn't find a `{{compat}}` duplicate. I'll look more closely.) (cc. @SnowMarble)

@queengooborg, All of these PRs are well-fixed and look fine, is it okay to merge them?

File: files/ko/web/css/_doublecolon_-webkit-scrollbar/index.md (hunk 2 above)
Comment (queengooborg) [reply files/ko/web/css/_doublecolon_-webkit-scrollbar/index.md:119]: All good, there's always some special case in these big refactoring projects that's unique to that one page/group of pages!

**Comment by queengooborg** [comment]
> Sorry for merging in [the last PR](https://github.com/mdn/translated-content/pull/11691#discussion_r1111375300) without asking. (+ I've looked all over... But didn't find a `{{compat}}` duplicate. I'll look more closely.)

Haha, it's not a problem at all, I'm just an English MDN author anyways!  😛   I think I speak for the entire MDN team when I say we're grateful for all your hard work maintaining the Korean localization of the MDN project!

> @queengooborg, All of these PRs are well-fixed and look fine, is it okay to merge them?

This is all ready to merge if that's what you're asking, yes!

**Pull request merged**